setup.py

```diff
@@ -29,9 +29,9 @@
     packages=find_packages(exclude=('library', 'bin')),
     include_package_data=True,
     install_requires=[
-        'ansible>=2.2.1',
+        'ansible>=2.3.2',
         'plumbum>=1.6.0',
-        'linchpin>=1.0.1'
+        'linchpin>=1.0.3'
```

Comment:
I'm in favor of depending on linchpin 1.0.2 since there is an issue with Beaker provisioning in 1.0.3:

https://github.com/CentOS-PaaS-SIG/linchpin/pull/358

Reply:
1.0.2 is still pinned to Ansible==2.2.whatever, and our playbooks have actually been dependent on 2.3 for a while now (per the issue this branch is mentioned from). Not sure there's a happy resolution to this, at the moment.

Reply:
Is it just that NTP task with the named block that depends on Ansible 2.3 or is there more?

Reply:
That's the only portion I'm presently aware of. But I'm not certain - are there any modules we use that you know of?